easybuild/easyconfigs/r/RAPSearch2/RAPSearch2-2.24-GCC-9.3.0.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GNU GPLv1
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'RAPSearch2'
+version = '2.24'
+
+homepage = 'https://omics.informatics.indiana.edu/mg/RAPSearch2'
+description = """
+RAPSearch stands for Reduced Alphabet based Protein similarity Search
+"""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://sourceforge.net/projects/rapsearch2/files']
+sources = ['%(name)s.24_64bits.tar.gz']
+checksums = ['85db4573f4c768b6c3c73bb44ff2611ba48dc6c8d188feb40f44bf7c55de36ce']
+
+install_cmd = './install'
```

Comment:
Actually,  I don't think this is called at all here at all (perhaps it's not used when you specify files to copy)? I doesn't build anything, and you are just copying the prebuilt binaries. This actually depends on boost, so it wouldn't build without it.
I think you want `build_cmd` here (despite the name of this script "install", it actually just builds)